conf/machine/colibri-imx7-emmc.conf

```diff
@@ -0,0 +1,31 @@
+#@TYPE: Machine
+#@NAME: Toradex Colibri iMX7 Dual 1GB (eMMC)
+#@SOC: i.MX 7Dual
+#@DESCRIPTION: Machine configuration for Toradex Colibri iMX7 SOM
+#@MAINTAINER: Stefan Agner <stefan.agner@toradex.com>
+
+MACHINEOVERRIDES =. "mx7:mx7d:"
+
+include conf/machine/include/imx-base.inc
+include conf/machine/include/tune-cortexa7.inc
+
+PREFERRED_PROVIDER_virtual/kernel ?= "linux-toradex"
+KERNEL_IMAGETYPE = "zImage"
+KERNEL_DEVICETREE += "imx7d-colibri-emmc-eval-v3.dtb imx7d-colibri-emmc-aster.dtb"
+
+# The kernel lives in a seperate FAT partition, don't deploy it in /boot/
+RDEPENDS_${KERNEL_PACKAGE_NAME}-base = ""
+
+PREFERRED_PROVIDER_u-boot ?= "u-boot-toradex"
+PREFERRED_PROVIDER_virtual/bootloader ?= "u-boot-toradex"
+PREFERRED_PROVIDER_u-boot-fw-utils ?= "u-boot-toradex-fw-utils"
+UBOOT_MACHINE ?= "colibri_imx7_emmc_defconfig"
+
+IMAGE_FSTYPES += "wic.gz tar.xz"
```

Comment:
The wic.gz is today already provided by imx-base.inc and could be deleted.

Reply:
Done.